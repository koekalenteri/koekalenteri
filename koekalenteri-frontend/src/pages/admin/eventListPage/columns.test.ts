import useEventListColumns from './columns'

jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: (k: string) => k}),
}))

jest.mock('../../recoil/judges', () => ({
  useJudgesActions: () => ({
    find: (id: number) => ({id, name: `Judge ${id}`}),
  }),
}))

describe('useEventListColumns', () => {
  it('should match snapshot', () => {
    expect(useEventListColumns()).toMatchSnapshot()
  })
})
