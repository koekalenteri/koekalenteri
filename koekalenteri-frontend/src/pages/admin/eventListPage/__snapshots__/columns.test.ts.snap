// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useEventListColumns should match snapshot 1`] = `
Array [
  Object {
    "align": "right",
    "field": "date",
    "headerName": "date",
    "hideable": false,
    "sortComparator": [Function],
    "valueFormatter": [Function],
    "valueGetter": [Function],
    "width": 120,
  },
  Object {
    "field": "eventType",
    "headerName": "event.eventType",
    "hideable": false,
    "minWidth": 100,
  },
  Object {
    "field": "classes",
    "flex": 1,
    "headerName": "event.classes",
    "minWidth": 100,
    "valueGetter": [Function],
  },
  Object {
    "field": "name",
    "flex": 1,
    "headerName": "event.name",
    "minWidth": 100,
  },
  Object {
    "field": "location",
    "flex": 1,
    "headerName": "event.location",
    "minWidth": 100,
  },
  Object {
    "field": "secretary",
    "flex": 1,
    "headerName": "event.secretary",
    "minWidth": 100,
    "valueGetter": [Function],
  },
  Object {
    "field": "official",
    "flex": 1,
    "headerName": "event.official",
    "minWidth": 100,
    "valueGetter": [Function],
  },
  Object {
    "field": "judges",
    "flex": 1,
    "headerName": "judgeChief",
    "minWidth": 100,
    "valueGetter": [Function],
  },
  Object {
    "align": "right",
    "field": "places",
    "headerName": "places",
    "valueGetter": [Function],
    "width": 80,
  },
  Object {
    "field": "state",
    "flex": 1,
    "headerName": "event.state",
    "hideable": false,
    "type": "string",
    "valueGetter": [Function],
  },
]
`;
